Azure.ResourceManager.Core/ResourceContainerBase.cs

```diff
@@ -8,77 +8,80 @@
 namespace Azure.ResourceManager.Core
 {
     /// <summary>
-    ///     Create known Container and Leaf scopes for ARM Containers
+    /// Base class for resource container
+    /// TODO:    Create known Container and Leaf scopes for ARM Containers
     ///     Think about how to extend known scope types in an extensible fashion (is it just adding them to the default, or is
     ///     it having scopes for all provider or consumer services?
     ///     For example, INetworkConsumer, IDatabaseConsumer, IEncryptionConsumer, IControlConsumer, ITriggerConsumer which
     ///     also allows you to attach at that scope? [AttachDatabase]
     /// </summary>
-    /// <typeparam name="TOperations">The class containing operations for the underlying resource</typeparam>
-    /// <typeparam name="TResource">The class containing properties for the underlying resource</typeparam>
+    /// <typeparam name="TOperations">The type of the class containing operations for the underlying resource</typeparam>
```

Comment:
lets add the space before and after the sentence along with "." at the end
this is in quite a few places but only marking one